test/bot.test.js

```diff
@@ -98,7 +98,7 @@ describe('Bot', function () {
     };
 
     bot.sendToIRC(message);
-    const expected = `<${message.author.username}> ${text}`;
+    const expected = `<${message.author.username.split('').join('\u200d')}> ${text}`;
```

Comment:
could add a simple helper function somewhere towards the top of the file that does the splitting?
i.e.

```js
const escapeUsername = username => username.split('').join('\u200d');
```